Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -857,19 +857,23 @@ void MainWindow::updateViewerBBox()
   const double xmax = bbox.xmax();
   const double ymax = bbox.ymax();
   const double zmax = bbox.zmax();
+
    //qDebug() << QString("Bounding box: (%1, %2, %3) - (%4, %5, %6)\n")
    //.arg(xmin).arg(ymin).arg(zmin).arg(xmax).arg(ymax).arg(zmax);
   qglviewer::Vec 
     vec_min(xmin, ymin, zmin),
-    vec_max(xmax, ymax, zmax);
+    vec_max(xmax, ymax, zmax),
+    bbox_center((xmin+xmax)/2, (ymin+ymax)/2, (zmin+zmax)/2);
   qglviewer::Vec offset(0,0,0);
-  for(int i=0; i<3; ++i)
-  {
-    if(std::log2(bbox.min(i)) > 13.0 )
+  double l_dist = (std::max)((std::abs)(bbox_center.x + viewer->offset().x),
+                      (std::max)((std::abs)(bbox_center.y + viewer->offset().y),
+                          (std::abs)(bbox_center.z + viewer->offset().z)));
+  if((std::log2)(l_dist) > 13.0 )
```

Comment:
**Note for next time** (no need to change the commit): only `min` and `max` have to be protected by a pair of parenthesis (that prevents the macro substitution). The reason are those two lines in `<stdlib.h>` in the MSVC standard library:
```C++
#define max(a,b)    (((a) > (b)) ? (a) : (b))
#define min(a,b)    (((a) < (b)) ? (a) : (b))
```

Without the protection against substitution of `max`, `std::max(x, y-1)`, for example, is turned into:
```C++
std::   (((x) > (y-1)) ? (x) : (y-1))
```
which leads to a parsing error.

No need to protect `abs` or `log2`, or anything else but `min` and `max`, as far as we known.